hcloud/load_balancer.go

```diff
@@ -119,6 +119,25 @@ type LoadBalancerTargetType string
 const (
 	// LoadBalancerTargetTypeServer is a target type which points to a specific server.
 	LoadBalancerTargetTypeServer LoadBalancerTargetType = "server"
+
+	// LoadBalancerTargetTypeLabelSelector is a target type which points to a label selector.
```

Comment:
That sentence doesn’t make sense. A load balancer cannot point to a label selector. 😉